apps/onboarding-functions/src/main/java/it/pagopa/selfcare/onboarding/functions/OnboardingFunctions.java

```diff
@@ -139,6 +146,60 @@ public void onboardingsOrchestrator(
         }
     }
 
+    /**
+     * This HTTP-triggered function starts the orchestration.
+     * Depending on the time required to get the response from the orchestration instance, there are two cases:
+     * * The orchestration instances complete within the defined timeout and the response is the actual orchestration instance output, delivered synchronously.
+     * * The orchestration instances can't complete within the defined timeout, and the response is the default one described in http api uri
+     */
+    @FunctionName("Notifications")
+    public HttpResponseMessage sendNotifications(
+            @HttpTrigger(name = "req", methods = {HttpMethod.POST}, authLevel = AuthorizationLevel.FUNCTION) HttpRequestMessage<Optional<String>> request,
+            @DurableClientInput(name = "durableContext") DurableClientContext durableContext,
+            final ExecutionContext context) {
+        context.getLogger().info("sendNotifications trigger processed a request.");
+
+        // Check request body
+        if (request.getBody().isEmpty()) {
+            return request.createResponseBuilder(HttpStatus.BAD_REQUEST)
+                    .body("\"Request body cannot be empty.")
+                    .build();
+        }
+
+        final String onboarding = request.getBody().get();
+        final String timeoutString = request.getQueryParameters().get("timeout");
+        DurableTaskClient client = durableContext.getClient();
+        String instanceId = client.scheduleNewOrchestrationInstance(SEND_ONBOARDING_NOTIFICATION, onboarding);
```

Comment:
you could avoid to use durable function only for calling one function....you could just use an http azure function

Reply:
agree